hub/executors/encoders/image/tfkeras-mobilenet_v2/manifest.yml

```
@@ -0,0 +1,9 @@
+name: tfkeras mobilenet_v2
+description: an image with the encoder of mobilenet.v2 from tfkeras
+author: Jina AI Dev-Team (dev-team@jina.ai)
```

@kashyapvad Last but not the least, it is our great honor to have your name listed here. ;)

```suggestion
author: Jina AI Dev-Team (dev-team@jina.ai)
```

---

The honor's mine :)